user_guide/data_management_data_audit.rst

```diff
@@ -3,8 +3,18 @@
 Data Audit
 ==========
 
-To see an out-of-the-box report of all the actions, performed with records of an
-:ref:`auditable entity <user-guide-entity-management-create-other>`, go to *System → Data Audit*.
+With the functionality, the user can see full history of changes made to any record of an 
+:ref:`auditable entity <user-guide-entity-management-create-other>` and the out-of-the-box 
+report of all such actions.
```

Comment:
Edit: With this functionality, users can see the full history of changes made to any record of an :ref:`auditable entity <user-guide-entity-management-create-other>`, as well as the out-of-the-box report of all such actions.